### PR TITLE
Fix/country profile ui

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/root/components/header.js [hunk 1]
```diff
@@ -148,6 +148,7 @@ function Header (props) {
                 <li>
                   <NavLink to='/' title={strings.headerMenuHomeTooltip} activeClassName='navbar-highlighted' exact>
                     <Translate stringId='headerMenuHome' />
+                    { strings.headerMenuHome }
```
Comment (batpad) [inline src/root/components/header.js:151]: This is causing Home to appear twice in the menu :)

**Comment by batpad** [comment]
This looks good to me! (Added a small catch about the Home in the Menu)

Added @imohkay as reviewer for usage of styles, etc.

**Comment by batpad** [comment]
@imohkay you can view this page locally with something like http://localhost:3000/countries/14#operations

File: src/root/components/header.js (hunk 1 above)
Comment (frozenhelium) [reply src/root/components/header.js:151]: Oops! Was only meant for testing. Thanks for catching it :+1:

**Comment by batpad** [comment]
@frozenhelium no objections to merging here.

**Comment by imohkay** [comment]
@frozenhelium @batpad this looks okay to me. I haven't compared the 3W page to what's on Zeplin in great detail. Do I need to do that?

**Comment by frozenhelium** [comment]
@imohkay It looks fairly similar. You can quickly check it if you want to

**Pull request merged**